sphinx/ext/imgconverter.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""
+    sphinx.ext.imgconverter
+    ~~~~~~~~~~~~~~~~~~~~~~~
+
+    Image converter extension for Sphinx
+
+    :copyright: Copyright 2007-2017 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+import subprocess
+
+from sphinx.errors import ExtensionError
+from sphinx.locale import _
+from sphinx.transforms.post_transforms.images import ImageConverter
+from sphinx.util import logging
+from sphinx.util.osutil import ENOENT, EPIPE, EINVAL
+
+if False:
+    # For type annotation
+    from typing import Any, Dict  # NOQA
+    from sphinx.application import Sphinx  # NOQA
+
+
+logger = logging.getLogger(__name__)
+
+
+class ImagemagickConverter(ImageConverter):
+    conversion_rules = [
+        ('image/svg+xml', 'image/png'),
+        ('application/pdf', 'image/png'),
+    ]
+
+    def is_available(self):
+        # type: () -> bool
+        """Confirms the converter is available or not."""
+        try:
+            ret = subprocess.call([self.config.image_converter, '-version'],
+                                  stdin=subprocess.PIPE, stdout=subprocess.PIPE)
+            if ret == 0:
+                return True
+            else:
+                return False
+        except (OSError, IOError):
+            logger.warning(_('convert command %r cannot be run.'
+                             'check the image_converter setting'),
+                           self.config.image_converter)
```

Comment:
Question is: how does user know what command (imagemagick) is required to use this extension?